doc/common/README.md

```diff
@@ -777,6 +777,13 @@ b. Правило добавляется с пометкой `\[Не автом
 
 c. Должна быть создана задача на автоматизацию правила.
 
+### 8.3. Локализация предложений
+
+a. Предложение должно быть оформлено на двух языках: русский и английский.
+
+b. При подаче предложения допускается использовать только один из них (русский).
```

Comment:
May be give an option for choosing russian or english?

Reply:
Does it make sense? Eventually we should describe solution using both languages. However why will we choose language we know less than our native for the primary description? It can be less readable due to lack of language skill. What purpose is?